release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -243,6 +243,31 @@ This enhancement adds the ability to configure global HTTP traffic compression o
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-configuring-router-compression_configuring-ingress[Using router compression].
 
+[id="ocp-4-10-networking-linuxptp"]
+==== Boundary clock and PTP enhancements to linuxptp services
+
+You can now specify multiple network interfaces in a `PtpConfig` profile to allow nodes running RAN vDU applications to serve as a Precision Time Protocol Telecom Boundary Clock (PTP T-BC). Interfaces configured as boundary clocks now also support PTP fast events.
+
+For more information, see xref:../networking/using-ptp.adoc#configuring-linuxptp-services-as-boundary-clock_using-ptp[Configuring linuxptp services as boundary clock].
+
+[id="ocp-4-10-networking-columbiaville"]
+==== Support for Intel 800-Series Columbiaville NICs
+
+Intel 800-Series Columbiaville NICs are now fully supported for interfaces configured as boundary clocks or ordinary clocks. Columbiaville NICs are supported in the following configurations:
+
+* Ordinary clock.
```

Comment:
Nit: probably could drop punctuation in these sentence fragment list items, but nbd.

Reply:
done